docs/multi-stage-query/index.md

```diff
@@ -33,7 +33,7 @@ This extension adds a [multi-stage query task engine for SQL](concepts.md#sql-ta
 [INSERT](concepts.md#insert) and [REPLACE](concepts.md#replace) statements as batch tasks. As an experimental feature,
 the task engine also supports running SELECT queries as batch tasks.
 
-Nearly all SELECT capabilities are available in the SQL task engine, with certain exceptions listed on the [Known
+Nearly all SELECT capabilities are available in the multi-stage query (MSQ) task engine, with certain exceptions listed on the [Known
```

Comment:
Could you please make the same edit in `api.md`? There's some very similar language there.